x-pack/plugins/saved_objects_tagging/public/components/base/tag_selector.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useMemo, useCallback } from 'react';
+import { EuiComboBox, EuiHealth, EuiHighlight, EuiComboBoxOptionOption } from '@elastic/eui';
+import { Tag } from '../../../common';
+
+type TagComboBoxOptions = EuiComboBoxOptionOption<Tag>;
+
+// we may need to add some props to pass down to EuiComboBox such as fullWidth or className
+// but it was not necessary for the initial usages.
```

Comment:
Would delete, the comment doesn't really add anything imo... We could need a lot of code but we don't because it's not necessary 🤷‍♂️